src/widget/paintable.cpp

```diff
@@ -16,35 +16,36 @@
 // static
 Paintable::DrawMode Paintable::DrawModeFromString(const QString& str) {
     if (str.compare("FIXED", Qt::CaseInsensitive) == 0) {
-        return FIXED;
+        return DrawMode::FIXED;
     } else if (str.compare("STRETCH", Qt::CaseInsensitive) == 0) {
-        return STRETCH;
+        return DrawMode::STRETCH;
     } else if (str.compare("STRETCH_ASPECT", Qt::CaseInsensitive) == 0) {
-        return STRETCH_ASPECT;
+        return DrawMode::STRETCH_ASPECT;
     } else if (str.compare("TILE", Qt::CaseInsensitive) == 0) {
-        return TILE;
+        return DrawMode::TILE;
     }
 
     // Fall back on the implicit default from before Mixxx supported draw modes.
     qWarning() << "Unknown DrawMode string in DrawModeFromString:"
                << str << "using FIXED";
-    return FIXED;
+    return DrawMode::FIXED;
 }
 
 // static
 QString Paintable::DrawModeToString(DrawMode mode) {
     switch (mode) {
-        case FIXED:
-            return "FIXED";
-        case STRETCH:
-            return "STRETCH";
-        case STRETCH_ASPECT:
-            return "STRETCH_ASPECT";
-        case TILE:
```

Comment:
these could be autogenerated if `DrawMode` was a Q_ENUM. Do you agree? Do we need to provide a stability guarantee for these strings?